src/lib/flow/include/sol-flow.h

```diff
@@ -171,6 +172,8 @@ int sol_flow_send_direction_vector_components_packet(struct sol_flow_node *src,
 
 int sol_flow_send_location_packet(struct sol_flow_node *src, uint16_t src_port, const struct sol_location *value);
 
+int sol_flow_send_timestamp_packet(struct sol_flow_node *src, uint16_t src_port, const time_t *value);
```

Comment:
Initially I was using an internal struct and it stayed as a pointer.
I'll change that.